portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path

```diff
@@ -260,7 +265,7 @@
 </tr>
 <tr>
 	<td>SELECT_BUTTON</td>
-	<td>//div[@class='criterion'][${key_indexField}]//div[contains(@class,'input-group-append')]</td>
+	<td>//div[@class='criterion']//div[contains(@class,'input-group-append')]</td>
```

Comment:
Hey @calvinctvo, we have two tests that need this index because for them we have more than one criterion added. I think you could pass the value of this index equal to 1 in your test so it doesn't affect others. What do you think?

- CanCreateDynamicSegmentWithIndividualCriteriaUsesBooleanAndDoesNotContain
- CanCreateDynamicSegmentWithIndividualCriteriaUsesIsKnownAndIsNot